app/views/register/new.html.haml

```diff
@@ -1,6 +1,9 @@
 = content_for :page_title, "New #{params[:register]}"
 = link_to 'Back', "/#{params[:register]}", class: 'link-back'
 
+- if flash.present?
```

Comment:
We should shorten this to `= render partial: 'register/shared/form-error-summary' if flash.present?` but I think this should be in the application layout file instead